### PR TITLE
deprecate: spec-generator

### PR DESCRIPTION
**Pull request by Malix-Labs**
## What type of PR is this?

- Chore

## Description

_Remove deprecated artifacts_

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,4 @@
 npm-debug.log
 _site
-spec-generator/node_modules
```
Comment (bamurtaugh) [inline .gitignore:3]: Do you have a pointer to why / when we should consider these deprecated?

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,4 @@
 npm-debug.log
 _site
-spec-generator/node_modules
-spec-generator/out
 devcontainer-index.json
-devcontainer-control-manifest.json
```
Comment (bamurtaugh) [inline .gitignore:6]: Why would we also remove this?

**Review by bamurtaugh — COMMENTED** [review]
Thanks so much for your contribution! Left a couple questions.

File: .gitignore (hunk 2 above)
Comment (Malix-Labs) [reply .gitignore:6]: The only thing changed in this line is an additional line (EOF)
Nothing has been removed
See https://github.com/devcontainers/devcontainers.github.io/pull/478/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L6-R4

Your mislead is understandable and due to GitHub bad formatting

**Comment by Malix-Labs** [comment]
Wait actually I don't remember doing this PR at all but if I did it there was a reason

Not that problematic anyway

**Pull request closed without merging**